sections/faqs/dom-two-classes.js

```diff
@@ -0,0 +1,13 @@
+import md from 'components/md'
+
+const TwoDomClasses = () => md`
+  ## Why does a DOM node have two classes?
```

Comment:
Can we change this to:

> Why do my DOM nodes have two classes?

otherwise it sounds like all DOM nodes are being referred to, regardless of whether they're mounted by styled components or not

Reply:
Sure, I agree it's better.